tests/unit/backend/corpora/common/entities/test_dataset.py

```diff
@@ -124,11 +123,17 @@ def test__create_ids__ok(self):
         self.assertIsNotNone(dataset)
         self.assertEqual(dataset_id, dataset.id)
 
-        self.assertEqual(artifact_ids, get_ids(dataset.artifacts))
-        self.assertNotEqual(["test_dataset_artifact_id"], get_ids(dataset.artifacts))
+        self.assertCountEqual(artifact_ids, [i.id for i in dataset.artifacts])
+        self.assertNotEqual(["test_dataset_artifact_id"], [i.id for i in dataset.artifacts])
 
-        self.assertEqual(deployment_directory_ids, get_ids(dataset.deployment_directories))
-        self.assertNotEqual(["test_dataset_artifact_id"], get_ids(dataset.deployment_directories))
+        self.assertCountEqual(deployment_directory_ids, [i.id for i in dataset.deployment_directories])
+        self.assertNotEqual(["test_dataset_artifact_id"], [i.id for i in dataset.deployment_directories])
 
-        self.assertEqual(contributor_ids, get_ids(dataset.contributors))
-        self.assertEqual(["test_contributor_id"], get_ids(dataset.contributors))
+        self.assertCountEqual(contributor_ids, [i.id for i in dataset.contributors])
+        self.assertCountEqual(["test_contributor_id"], [i.id for i in dataset.contributors])
```

Comment:
Are both of these asserts testing effectively the same thing - one item in the list?

Reply:
yep, will removed one.